tests/testthat/_snaps/table_02.md

```diff
@@ -23,7 +23,7 @@
         MULTIPLE                                            0                 1 (0.7%)                0                 1 (0.2%)     
         NATIVE HAWAIIAN OR OTHER PACIFIC ISLANDER           0                 1 (0.7%)                0                 1 (0.2%)     
       Ethnicity                                                                                                                      
-        NOT REPORTED                                     6 (4.5%)            10 (7.5%)            11 (8.3%)            27 (6.8%)     
+         NOT REPORTED                                    6 (4.5%)            10 (7.5%)            11 (8.3%)            27 (6.8%)     
```

Comment:
The changes in table 2 and 21 snapshots are caused by your local formatters version (you have the trimming branch loaded), so these changes can be reverted.